pytensor/link/pytorch/dispatch/basic.py

```diff
@@ -24,8 +24,11 @@
 
 
 @singledispatch
-def pytorch_typify(data, **kwargs):
-    raise NotImplementedError(f"pytorch_typify is not implemented for {type(data)}")
+def pytorch_typify(data, dtype=None, **kwargs):
+    if dtype is None:
+        return data
+    else:
+        return torch.tensor(data, dtype=dtype)
```

Comment:
We change this approach. You need to dispatch on the RNG type and decide what to do with it. The base-cass is to raise